packages/snippet-bot/test/violations.ts

```diff
@@ -25,13 +29,74 @@ import {
 
 import * as snippetsModule from '../src/snippets';
 import {Snippets} from '../src/snippets';
+import {ApiLabels} from '../src/api-labels';
+import * as apiLabelsModule from '../src/api-labels';
 import {resolve} from 'path';
 import assert from 'assert';
 import {describe, it} from 'mocha';
 import * as sinon from 'sinon';
 
 const fixturesPath = resolve(__dirname, '../../test/fixtures');
 
+describe('checkProductPrefixViolations', async () => {
```

Comment:
no need for `async` here 👍 I think you can use `async`, if you do some async initialization:

```
desceribe('checkProductPrefixViolations', async () => {
  const sharedConfig = await loadConfig();
});
```

But I'd only do this if needed.

Reply:
```suggestion
describe('checkProductPrefixViolations', () => {
```

Reply:
Sure, removed!